src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
+  "block.pipez.item_pipe": "Item Röhre",
+  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
+  "block.pipez.energy_pipe": "Energie Röhre",
+  "block.pipez.universal_pipe": "Universele Röhre",
+  "block.pipez.gas_pipe": "Gas Röhre",
+  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
+  "item.pipez.improved_upgrade": "Verbesserte Röhre Verbesserung",
+  "item.pipez.advanced_upgrade": "Erweiterte Röhre Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Röhre Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Röhre Verbesserung",
+  "item.pipez.wrench": "Röhren Zange",
+  "item.pipez.filter_destination_tool": "Filter Ziel Gerät",
+  "tooltip.pipez.wrench": "Klicke auf eine Röhre um sie einzustellen",
+  "tooltip.pipez.distribution": "Verteilung: %s",
+  "tooltip.pipez.distribution.nearest": "Nächste zuerst",
+  "tooltip.pipez.distribution.furthest": "Weiteste zuert",
+  "tooltip.pipez.distribution.round_robin": "Rundlauf",
+  "tooltip.pipez.distribution.random": "Zufällig",
+  "tooltip.pipez.redstone_mode": "Rotstein Modus: %s",
+  "tooltip.pipez.redstone_mode.ignored": "Ignoriert",
+  "tooltip.pipez.redstone_mode.off_when_powered": "Aus Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.on_when_powered": "An Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.always_off": "Immer Aus",
+  "tooltip.pipez.filter_mode": "Filter Modus: %s",
+  "tooltip.pipez.filter_mode.whitelist": "Erlaubt",
+  "tooltip.pipez.filter_mode.blacklist": "Nicht Erlaubt",
+  "tooltip.pipez.filter.not": "Nicht",
+  "tooltip.pipez.filter.accepts_tag": "Alle %s",
+  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
```

Comment:
Inconsistent capitalization
Also not a proper sentence

Reply:
Didnt know the exact usage of it. Mind helping me out?

Reply:
"Item IDs oder Tags mit #"